app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
Assuming the team is OK with this strategy, this should come after the initial check for a valid mime type in the datastream.

As-is, it would preemptively assign the content type header for an image before even checking if we have a mime type from the technical metadata.